.gitignore

```diff
@@ -3,689 +3,3 @@ node_modules
 out
 dist/extension.js
 dist/extension.js.map
-.vscode-test/vscode-1.42.1/chrome_100_percent.pak
```

Comment:
After deleting these lines, nothing extra was showing in my pending changes list. Maybe a hangover from previous work on this repo?
If stuff under .vscode-test still needs to be ignored then maybe it can be done with a pattern.

Reply:
lets leave it at the moment